ext/hal/st/stm32cube/Kbuild

```diff
@@ -15,6 +15,8 @@ obj-y += stm32f3xx/drivers/src/stm32f3xx_hal.o
 obj-y += stm32f3xx/drivers/src/stm32f3xx_hal_rcc.o
 obj-$(CONFIG_CLOCK_CONTROL_STM32_CUBE) += stm32f3xx/drivers/src/stm32f3xx_ll_utils.o
 obj-$(CONFIG_SERIAL_HAS_DRIVER) += stm32f3xx/drivers/src/stm32f3xx_hal_uart.o
+obj-$(CONFIG_I2C) += stm32f3xx/drivers/src/stm32f3xx_ll_i2c.o
+obj-$(CONFIG_I2C) += stm32f3xx/drivers/src/stm32f3xx_ll_rcc.o
```

Comment:
I know we did this for the other STM32 I2C devices, but this really should be something like obj-$(CONFIG_I2C_STM32_V2) += ...

Reply:
CONFIG_I2C selects CONFIG_I2C_STM32_V2 in Kconfig.series. I don't really see any reason why we need to specify here the I2C controller version. If it is a must, I'll change it and I'll create a PR for the other families.